package.json

```diff
@@ -26,6 +26,7 @@
   },
   "private": true,
   "workspaces": [
+    "external-api-cache",
```

Comment:
not for this PR, but I'm starting to wonder if we should create an `apis/*` folder or similar (some workspaces definitions group apis and app under `apps/*`, similar to `packages/*`

Reply:
Or consolidate all APIs in a single service. Let's keep thinking about this. The current state is not ideal at all.